src/styles/Inbox.css

```diff
@@ -12,15 +12,21 @@
 .recieved {
   background-color: var(--convene-gray);
   color: var(--convene-blue);
+  word-wrap: break-word;
+  overflow-wrap: break-word;
 }
 
```

Comment:
could move the word-wrap and overflow-wrap from "sent" and "received" to "message" but no big deal

Reply:
You right! I was just doing changes so fast didn't  realize that